pkg/apiserver/compression_logs.go

```diff
@@ -0,0 +1,50 @@
+package apiserver
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/justtrackio/gosoline/pkg/log"
+)
+
+const (
+	requestSizeFields           = "goso.request.sizeFields"
+	requestCompressedSizeFields = "goso.request.compressed.sizeFields"
+	responseSizeFields          = "goso.response.sizeFields"
+)
+
+type sizeData struct {
+	size *int
+}
+
+type encodedSizeData struct {
+	sizeData
+	contentEncoding string
+}
+
+func getRequestSizeFields(ginCtx *gin.Context) log.Fields {
+	result := log.Fields{}
+
+	requestSizeFields, found := ginCtx.Get(requestSizeFields)
+	if found && requestSizeFields != nil {
+		if data, ok := requestSizeFields.(sizeData); ok {
+			result["request_size"] = *data.size
+		}
+	}
+
+	requestCompressedSizeFields, found := ginCtx.Get(requestCompressedSizeFields)
+	if found && requestCompressedSizeFields != nil {
+		if data, ok := requestCompressedSizeFields.(encodedSizeData); ok {
+			result["request_compression"] = data.contentEncoding
+			result["request_uncompressed_size"] = *data.size
+		}
+	}
+
+	responseSizeFields, found := ginCtx.Get(responseSizeFields)
+	if found && responseSizeFields != nil {
+		if data, ok := responseSizeFields.(encodedSizeData); ok {
+			result["response_compression"] = data.contentEncoding
+			result["uncompressed_bytes"] = *data.size
```

Comment:
```suggestion
			result["response_uncompressed_size"] = *data.size
```

Reply:
Hmm, we already have a field `bytes` which records the raw size (i.e., compressed if compression was applied) of a response, should I also rename that field then? I would like to keep the similarly named.  `gosoline/pkg/apiserver/middleware_logger.go:40`

Reply:
i see your point of keeping things named similarly. Since bytes is more speaking than size, i would actually prefer to have it say "bytes" everywhere. Could you adjust that?